src/dotnet/commands/dotnet-remove/dotnet-remove-package/LocalizableStrings.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Remove.PackageReference
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET Remove Package reference Command";
+
+        public const string AppDescription = "Command to remove package reference";
+
+        public const string AppHelpText = "Package references to remove";
+
+        public const string SpecifyExactlyOnePackageReference = "Please specify one package reference to remove";
```

Comment:
Should this say: Please specify only one package reference to remove? Also, add dots (.) to the sentences above.

@blackdwarf To review the text here.